types/rpi-gpio/index.d.ts

```diff
@@ -0,0 +1,136 @@
+// Type definitions for rpi-gpio 2.1
+// Project: https://github.com/JamesBarwell/rpi-gpio.js#readme
+// Definitions by: Giles Roadnight <https://github.com/Roaders>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.0
+
+/// <reference types="node" />
+import { EventEmitter } from 'events';
+
+type MODE = 'mode_rpi' | 'mode_bcm';
+type DIR = 'in' | 'out' | 'low' | 'high';
```

Comment:
there is really no reason why this cannot be a full word with explicit meaning
```suggestion
type PinDirection = 'in' | 'out' | 'low' | 'high';
```
https://github.com/JamesBarwell/rpi-gpio.js#setupchannel--direction-edge-callback
note that author of package seems to use explicit terms:
https://github.com/JamesBarwell/rpi-gpio.js/blob/6dd19bf1adf8ba3ac48f13bdc1aa60f4d05341ee/rpi-gpio.js#L90-L94